server/api/urls.py

```diff
@@ -17,14 +17,17 @@
 
 from django.contrib import admin
 from django.urls import path, include
+from bingo import views
 
 from rest_framework_simplejwt.views import (
     TokenObtainPairView,
     TokenRefreshView,
 )
+
 urlpatterns = [
-    path("admin/", admin.site.urls),
+    path('admin/', admin.site.urls),
     path("api/healthcheck/", include(("api.healthcheck.urls"))),
     path('api/token/', TokenObtainPairView.as_view(), name='token_obtain_pair'),
     path('api/token/refresh/', TokenRefreshView.as_view(), name='token_refresh'),
+    path('register/', views.register_user, name='register_user'),
```

Comment:
Can you change the path to be ``api/register/`` instead? Or even better, move this path to a new ``urls.py`` in the bingo folder and import it to here.